practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/controller/PractitionerController.java

```diff
@@ -50,6 +52,12 @@ public PractitionerController(EntityService entityService) {
         return ResponseEntity.status(CREATED).body(new Response(practitionerId));
     }
 
+    @PostMapping(path = "/link", consumes = APPLICATION_JSON_VALUE, produces = APPLICATION_JSON_VALUE)
```

Comment:
1. no need for consumes/produces since it is defined at the class level.
2. can we do "link" under the specific practitioner just like we do role assignments: /practitioner/{id}/link?